### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Screen Reader: In scan mode NVDA/JAWS is reading “Frame Frame Frame” when down arrow is pressed after “more options” button

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/vscode-mssql#1229]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_mssqlforVSCode; #MAS1.3.1;  #C+E-H; #mssqlforVSCode; #ScreenReader; #NVDA; #JAWS

**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.18362 Build 18362)
Screen reader: NVDA, JAWS

**MAS Violated:** MAS 1.3.1

**Note: In JAWS while navigating to the to “Save as CSV” link and to “Line 1”link JAWS is reading “Frame Frame Frame”**

**Repro Steps:**
1.       Open Visual Studio Code in your system
2.       Navigate to "Extensions"   and install "SQl Server (mssql)" extension
3.       Press F1 and select MS SqL :   Connect and connect to any sql server
4.       Write code (SELECT * FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""    
5.       Start NVDA/JAWS  
6.       Tab till “more options” button and press down arrow in scan mode to read the non-text content   
7.        Verify In scan mode if NVDA/JAWS is reading “Frame Frame Frame” when down arrow is pressed after “more options” button

**Actual Result:** 
In scan mode NVDA/JAWS is reading “Frame Frame Frame” when down arrow is pressed after “more options” button

**Expected Result:** 
Screen reader should not read the information which is not present on the screen. Screen reader should read only “Results Ctl+Alt+R”

**User Impact:** 
Screen Reader users will be provided with wrong information if the screen readers read the content that is not present on the screen

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_1229_In scan mode NVDA is reading “Frame Frame Frame” when pressed down arrow after “more options” button](https://user-images.githubusercontent.com/41109135/56639940-77eeb580-668f-11e9-9ecc-37e08823d0c3.png)

![Bug_1229_jaws reading frame readonly at save as csv file link](https://user-images.githubusercontent.com/41109135/56658936-84880380-66b9-11e9-8d7a-cd9e2d1e173c.png)

**Comment by alanrenmsft** [comment]
issue fixed now, if you keep pressing down arrow, narrator will read: Virtual Document ->Query Output Document Host -> Query Output Document

'virtual document' layer is added by vscode

**Comment by fsteffi** [comment]
#closed;
![image](https://github.com/microsoft/vscode-mssql/assets/24580570/29462b17-dc34-4f6b-9d7c-705951eb008b)

Version: 1.79.0-insider (user setup)
Commit: fa8eefda18bf099d82b58453f8fe5d3010196fa0
Date: 2023-05-31T03:06:45.096Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370
